includes/rest-api.php

```diff
@@ -318,17 +320,46 @@ function pmpro_rest_api_get_has_membership_access($request) {
 		 */
 		function pmpro_rest_api_change_membership_level( $request ) {
 			$params = $request->get_params();
-			$user_id = isset( $params['user_id'] ) ? intval( $params['user_id'] ) : null;
-			$level_id = isset( $params['level_id'] ) ? intval( $params['level_id'] ) : null;
+			$user_id = isset( $params['user_id'] ) ? (int) $params['user_id'] : null;
+			$level_id = isset( $params['level_id'] ) ? (int) $params['level_id'] : null;
 			$email = isset( $params['email'] ) ? sanitize_email( $params['email'] ) : null;
+			$create_user = isset( $params['create_user'] ) ? (bool) $params['create_user'] : false;
```

Comment:
This might be a good idea which handles a few more cases

```suggestion
			$create_user = isset( $params['create_user'] ) ? filter_var( $params['create_user'], FILTER_VAR_BOOLEAN ) : false;
```